libs/k6/src/common/token.js

```
@@ -0,0 +1,134 @@
+let cachedTokensIssuedAt = {};
+
+/**
+ * Function to generate a cache key based on the token type and options.
+ * @param {*} tokenType 
+ * @param {*} tokenOptions 
+ * @returns 
+ */
+function getCacheKey(tokenType, tokenOptions) {
+    var cacheKey = `${tokenType}`;
+    for (const key in tokenOptions) {
+        if (tokenOptions.hasOwnProperty(key)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)